priv/tailwind.config.js.eex

```diff
@@ -3,11 +3,11 @@
 // See the Tailwind configuration guide for advanced usage
 // https://tailwindcss.com/docs/configuration
 
+const baseConfig = require("./tailwind.base.config");
```

Comment:
What I was mentioning in [this other comment](https://github.com/BeaconCMS/beacon/pull/542/files#r1667019785)